internal/analytics/persist_test.go

```
@@ -3,21 +3,23 @@ package analytics
+	"path/filepath"
 	"testing"
 	"time"
 
 	"github.com/stretchr/testify/require"
 )
 
 func TestPersistAcrossConnections(t *testing.T) {
-	tmpDir, err := os.MkdirTemp("", "dolt-persist-*")
+	tmpDir, err := os.MkdirTemp("", "persist-*")
 	require.NoError(t, err)
 	defer os.RemoveAll(tmpDir)
```

golangci-lint (2.12.2)

[error] Error return value of `os.RemoveAll` is not checked

(errcheck)